dcs/a10c_presets.py

```diff
@@ -0,0 +1,88 @@
+import dcs.lua as lua
+
+
+class A10CPresets:
+    def __init__(self):
+        self.presets: dict[int, int] = {
+            1: 305000000,
+            2: 264000000,
+            3: 265000000,
+            4: 256000000,
+            5: 254000000,
+            6: 250000000,
+            7: 270000000,
+            8: 257000000,
+            9: 255000000,
+            10: 262000000,
+            11: 259000000,
+            12: 268000000,
+            13: 269000000,
+            14: 260000000,
+            15: 263000000,
+            16: 261000000,
+            17: 267000000,
+            18: 251000000,
+            19: 253000000,
+            20: 266000000,
+        }
+
+    def load_dict(self, data):
+        presets = data.get("settings", {}).get("presets", {})
+        for chan, freq in presets.items():
```

Comment:
I guess this is a bit superflous? Could we just assign self.presets to the dict from the data?

Reply:
should work, until we need/want a deep copy